Demo/Demo/Models/Order.swift

```diff
@@ -36,9 +36,9 @@ struct Order: Codable, Equatable {
 
     struct Vault: Codable, Equatable {
         
-        let id: String
+        let id: String?
```

Comment:
Should this change be in this PR? It looks to be unrelated.

Reply:
When I switched from pointing to `next-major-version` from `main`, there were several conflicts that I resolved. I checked, and this change, along with others that you commented on, already exist on `next-major-version`. I didn't pull in the latest from `main` into `next-major-version` branch before creating this new branch `fix/button-edges-name`. Are you ok with me merging these changes into `next-major-version`?